install/installer/pkg/common/common.go

```diff
@@ -29,8 +29,10 @@ import (
 
 func DefaultLabels(component string) map[string]string {
 	return map[string]string{
-		"app":       AppName,
-		"component": component,
+		"app":              AppName,
+		"component":        component,
+		"kots.io/app-slug": AppName,
+		"kots.io/backup":   "velero",
```

Comment:
Some explanation/justification would be handy here.
At first glance this doesn't sit right that we'd mix particularities of one installation path into all of the others.

Reply:
This is still in draft and I'm playing with things at the moment. I'll let you know when it's ready for review - I'm not even sure this will be in the final PR